src/shims/env.rs

```diff
@@ -310,6 +338,27 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         }
     }
 
+    #[allow(non_snake_case)]
+    fn SetCurrentDirectoryW (
+        &mut self,
+        path_op: OpTy<'tcx, Tag> // LPCTSTR
+    ) -> InterpResult<'tcx, i32> {
```

Comment:
According to https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-setcurrentdirectory, this returns a `BOOL`. Is `BOOL` rally `i32` on Windows?

Reply:
Ah yes, `BOOL = i32` is even noted in the Windows shims file. Still, this is worth repeating in a comment here.